packages/baremetal-operator/charts/values.yaml

```diff
@@ -28,7 +28,7 @@ images:
   baremetalOperator:
     repository: registry.opensuse.org/isv/suse/edge/metal3/containers/images/baremetal-operator
     pullPolicy: IfNotPresent
-    tag: "0.9.1"
+    tag: "0.9.1.1"
```

Comment:
If we plan to release BMO from Factory similar to the IPA downloader image we may want to use the same https://build.opensuse.org/project/show/isv:SUSE:Edge:Containers project - in which case the repository above should be `registry.opensuse.org/isv/suse/edge/containers/images/baremetal-operator`